keyboards/cest73/tkm/config.h

```diff
@@ -41,10 +41,13 @@
 #define LED_NUM_LOCK_PIN F6
 #define LED_CAPS_LOCK_PIN F5
 #define LED_SCROLL_LOCK_PIN F7
-#define LED_PIN_ON_STATE 0
+#define LED_PIN_ON_STATE 1
 //TODO change the E6 pin to either B5, B6, B7 or C6 to utilize hardware PWM on a future PCB revision
 #define BACKLIGHT_PIN E6
 
+// mouse keys acceleration mode
+#define MK_COMBINED
+
```

Comment:
We generally do not like this being at the keyboard level, and instead should be migrated to a keymap level `keyboards/cest73/tkm/keymaps/default/config.h`
```suggestion
```